### PR TITLE
Use the newest HFSTOL during Search

### PR DESCRIPTION
**Pull request by Madoshakalaka**
- The newest hfstol file is used
- test database is rebuilt with the newest hfstol and the newest crkeng.xml
- pipenv update

### CODE REVIEW

File: CreeDictionary/API/models.py [hunk 1]
```diff
@@ -393,10 +394,12 @@ def fetch_lemma_by_user_query(user_query: str, **kwargs) -> "CreeAndEnglish":
...
                 # notice Err/Orth tags needs to be stripped because it makes our generator generate un-normatized forms
+                normatized_analysis = analysis.replace("+Err/Orth", "")
                 normatized_form_for_analysis = [
-                    *normative_generator_foma.generate(
-                        analysis.replace("+Err/Orth", "")
-                    )
+                    form
+                    for form in normative_generator.feed_in_bulk_fast(
+                        [normatized_analysis]
+                    )[normatized_analysis]
```
Comment (Madoshakalaka) [inline CreeDictionary/API/models.py]: what makes more sense is `normative_generator.feed(normatized_analysis)`, but sadly `.feed` from `hfstol` python package freezes with some wordforms. https://github.com/Madoshakalaka/hfstol/issues/1

File: CreeDictionary/API/models.py [hunk 2]
```diff
@@ -357,12 +360,10 @@ def fetch_lemma_by_user_query(user_query: str, **kwargs) -> "CreeAndEnglish":
         # 2. definition containment of the query word
         cree_results: Set[CreeResult] = set()
 
-        # utilize the spell relax in descriptive_analyzer
-        # TODO: use shared.descriptive_analyzer (HFSTOL) when this bug is fixed:
-        # https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/issues/120
```
Comment (Madoshakalaka) [inline CreeDictionary/API/models.py]: I released `hfstol-1.2.11` for #120, hopefully the bug is fixed
Screenshot from 2020-03-10 21-07-45

Comment (Madoshakalaka) [reply CreeDictionary/API/models.py]: good news: the ci passes

File: cypress/integration/regressions.spec.js [hunk 3]
```diff
@@ -99,15 +99,6 @@ context('Regressions', () => {
       .should('contain', 'atâhk')
   })
 
-  // https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/issues/176
-  describe('should show idiomatic lemmas', () => {
-    it('The Cree word ayiwinis should give you ayiwinisa as lemma', () => {
```
Comment (Madoshakalaka) [inline cypress/integration/regressions.spec.js]: the new hfstol gives "ayiwin" as the lemma, as the linguists decide "ayiwinisa" is not idiomatic enough.

File: cypress/integration/regressions.spec.js [hunk 4]
```diff
@@ -129,13 +120,6 @@ context('Regressions', () => {
     )
   })
 
-  it('The Cree word ayiwinis should give you ayiwinisa as lemma', () => {
```
Comment (Madoshakalaka) [inline cypress/integration/regressions.spec.js]: these "idiomatic lemma" test doesn't make sense anymore, so we can just delete them. We had the issue cuz we used a lemma_tags.tsv as our lemma form guideline, (which say, decides the lemma of NI words should be N+I+Sg). The importing process has been changed and the lemma form depends solely on the fst we have.

File: CreeDictionary/API/models.py (hunk 1 above)
Comment (eddieantonio) [reply CreeDictionary/API/models.py]: ¯\_(ツ)_/¯

**Pull request merged**